library/lib_gui_box.ks

```diff
@@ -0,0 +1,43 @@
+// This file is distributed under the terms of the MIT license, (c) the KSLib team
+// This file is based on lib_window.ks from akrOS by akrasuski1.
+
+function draw_custom_gui_box {
+	parameter
+		x, y, w, h,
+		horizontal_char,
+		vertical_char,
+    corner_char.
+
+  // to-discuss: should we sanitize the input?
```

Comment:
I will work on adding this into it later this evening (might end up as tomorrow)